kv/txn_coord_sender.go

```diff
@@ -277,37 +277,24 @@ func (tc *TxnCoordSender) startStats() {
 			}
 
 			num := len(curStats.durations)
-			dMax, err := stats.Max(curStats.durations)
-			if err != nil {
-				panic(err)
```

Comment:
I don't think we should be ignoring these; the error semantics may change in a future version of `stats`, and the panics are harmless if we're never hitting them in practice. Other changes LGTM